lib/SVF-FE/LLVMModule.cpp

```diff
@@ -145,10 +148,64 @@ void LLVMModuleSet::build()
             BasicBlock *exitBB =  const_cast<BasicBlock*>(LLVMUtil::getFunExitBB(func));
             svffun->setReachableBBs(reachableBBs);
             svffun->setExitBB(exitBB);
+            
+            //process and stored dt & df
+            DominatorTree dt;
+            DominanceFrontier df;
+            dt.recalculate(*svffun->getLLVMFun());
+            df.analyze(dt);
+            PostDominatorTree pdt = PostDominatorTree(*(func->getLLVMFun()));
+            Map<const BasicBlock*,Set<const BasicBlock*>> & dfBBsMap = svffun->getDomFrontierMap();
+            for (DominanceFrontierBase::const_iterator dfIter = df.begin(), eDfIter = df.end(); dfIter != eDfIter; dfIter++)
+            {
+                const BasicBlock* keyBB = dfIter->first;
+                const std::set<BasicBlock *>& domSet = dfIter->second;
+                Set<const BasicBlock*>& valueBasicBlocks = dfBBsMap[keyBB];
+                for (std::set<BasicBlock*>::iterator domSetIter = domSet.begin(), eDomSet = domSet.end(); domSetIter !=eDomSet; domSetIter++)
+                {
+                    BasicBlock* bbValue = *domSetIter;
+                    valueBasicBlocks.insert(bbValue);
+                }
+            }
+            for (Function::const_iterator bit = svffun->getLLVMFun()->begin(), ebit = svffun->getLLVMFun()->end(); bit != ebit; ++bit)
+            {   
+                const BasicBlock *bb = &*bit;
+                if(DomTreeNode *dtNode = dt.getNode(const_cast<BasicBlock*>(bb)))
+                {
+                    DomTreeNode::iterator DI = dtNode->begin();
+                    if (DI != dtNode->end()){
+                        for (DomTreeNode::iterator DI = dtNode->begin(), DE = dtNode->end(); DI != DE; ++DI)
+                        {
+                            BasicBlock* dtBB = (*DI)->getBlock();
+                            svffun->getDomTreeMap()[bb].insert(dtBB);
```

Comment:
`... .insert((*DI)->getBlock())`

Less noise since the expressions is 1) simple, and 2) never used again.